hubblestack/extmods/modules/conf_publisher.py

```diff
@@ -0,0 +1,46 @@
+
+import logging
+import hubblestack.splunklogging
+import copy
+import time
+
+log = logging.getLogger(__name__)
+
+def publish():
```

Comment:
Could you add some docstrings to these functions with some basic documentation about what they do? It's good to be consistent and document wherever possible.